### PR TITLE
Gh8 9

### PR DESCRIPTION
**Pull request by ayurchen**
Here are two commits:
  - makes apply_cb() return and to_execute_end() pass an int error code to provider.
  - adds assign_read_view() call

### CODE REVIEW

File: wsrep_api.h [hunk 1]
```diff
@@ -720,6 +719,18 @@ struct wsrep {
     wsrep_status_t (*recv)(wsrep_t* wsrep, void* recv_ctx);
 
   /*!
+   * @brief Tells provider that a given writeset has a read view associated
+   *        with it.
+   *
+   * @param wsrep  provider handle
+   * @param handle writeet handle
+   * @param rv     read view astablished by the caller or if NULL, by provider
```
Comment (temeo) [inline wsrep_api.h]: Fix typo astablished -> established.

Comment (temeo) [inline wsrep_api.h]: And I don't understand the description. How does this function call associates a read view with some given write set?

**Comment by ayurchen** [comment]
Pushed typos fix and tried to make description clearer, however I'm not sure what was wrong with it.

**Comment by temeo** [comment]
Looks ok to merge.

**Pull request merged**